korgan/claim_state_duty.py

```
@@ -0,0 +1,352 @@
+    verified = "\n".join(str(item) for item in research.verified_claims or [])
+    return bool(_CONSUMER_SOURCE_RE.search(verified))
+
+
+def _article668_exemption(case_context: str, draft: ClaimDraft) -> StateDutyDecision | None:
+    claimant = _claimant_context(case_context, draft)
+    relief = "\n".join([draft.title or "", *[str(item) for item in draft.requests or []]])
+
+    if _DISABILITY_RE.search(claimant):
+        return StateDutyDecision(
+            mode="exempt_disability",
+            line="0 тенге (освобождение от уплаты государственной пошлины; пункт 13 статьи 668 Налогового кодекса РК)",
```

Ruff (0.16.2)

[warning] String contains ambiguous `Р` (CYRILLIC CAPITAL LETTER ER). Did you mean `P` (LATIN CAPITAL LETTER P)?

(RUF001)